api/pkg/imagebuilder/imagebuilder_test.go

```diff
@@ -1052,11 +1052,13 @@ func Test_kanikoBuilder_imageRefExists(t *testing.T) {
 	type args struct {
 		imageName string
 		imageTag  string
+		retry     int
```

Comment:
is there a way to create test to simulate retry cases? if not possible add case where `retry` value greater than 0 should be sufficient